dictionaries/en-US.aff

```diff
@@ -0,0 +1,205 @@
+SET UTF-8
```

Comment:
what is all of this for?

Reply:
cargo copy cult from API repo. But to make a guess, this controls the encoding type of the dictionary that is used to spell check the API.

Reply:
can we avoid adding a new top level directory somehow?

Reply:
two options:
- can place dictionary in operator repo
- can remove spell-checking from operator's protofiles.

We sadly can't place the dict in the container - as folks would not be able to build with `BUILD_WITH_CONTAINER=0`.

Reply:
I am a bit confused why we need it at all since the operator repo didn't have it? but if we do can we put it under operator folder?

Reply:
I think it adds professional value - if someone comes along and cleans up the spelling errors and files. Even so, you are right, the dictionary isn't globally usable for the repo, so I will move it to the operator directory.

Reply:
should be addressed. If so, please click resolve 👍 

Cheers
-steve